support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYou.jsx

```diff
@@ -57,12 +61,34 @@ function ContributionThankYou(props: PropTypes) {
     <div className="thank-you__container">
       <div className="gu-content__form gu-content__form--thank-you">
         {props.contributionType !== 'ONE_OFF' ? (
-          <section className="confirmation">
-            <h3 className="confirmation__title">
+          <section className="contribution-thank-you-block">
+            <h3 className="contribution-thank-you-block__title">
               {`${directDebitHeaderSuffix}Look out for an email within three business days confirming your ${getSpokenType(props.contributionType)} recurring payment${directDebitMessageSuffix}`}
             </h3>
           </section>
         ) : null}
+        {!props.isSignedIn ?
+          <section className="contribution-thank-you-block">
+            <h3 className="contribution-thank-you-block__title">
+              Stay signed in to The Guardian
+            </h3>
+            <p className="contribution-thank-you-block__message">
+              As a valued contributor, we want to ensure you are having the best experience on our site.
+              To stop seeing requests for support at the bottom of articles, or in pop-up banners, please sign in
+              on each of the devices you use to access The Guardian – mobile, tablet, laptop or desktop.
+            </p>
+            <Button
+              aria-label="Sign into The Guardian"
+              appearance="secondary"
+              onClick={
+                () => {
+                  trackComponentClick(`sign-into-the-guardian-link-${props.contributionType}`);
```

Comment:
hopefully we could also get this info from pageview data on profile.theguardian.com/signin, where the referrer is the contributions thank you page. though that wouldn't distinguish one-off/recurring

just flagging for the future - this extra tracking is great for now as we're rolling this out but I'm always keen for links to be links if they can be

Reply:
yeah that's a good point. I think tracking could also be achieved with query params but i'm not that familiar with that method- in particular where that data ends up. Something for me to look into.